### PR TITLE
Add function:

### PR DESCRIPTION
**Pull request by zkx741481546**
1.draw 1 dimension picture
2.customize pic save path

changed:
keract/keract.py ---display_activations()
added param dim for dimension, only support 1 or 2
added dir for path to save picture, example:'pic/step1/'

### CODE REVIEW

**Comment by qwertpi** [comment]
LGTM but updating the docstrings to reflect the new parameters would be a good idea

**Comment by zkx741481546** [comment]
I just finished a new version that support draw no-channel data into 1D picture. I will change into Plotly or some other visualize package next version, I hope to make it faster. If things go well, I will send a new pull request. Kercat is the life-savior!

**Comment by qwertpi** [comment]
You can probably replace your data_format parameter with keras.backend.image_data_format(). The only thing with plotly is it is an extra dependency that people are less likely to already have installed than matplotlib

**Comment by zkx741481546** [comment]
I just finished a new version that support none-channel data into 1D picture, I had already pushed it, and I am planning changing matplotlib into Plotly and test if it’s faster than matlplotlib. If it works fine, I will send another pull request. This package is a life-savior!

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

________________________________
发件人: qwertpi <notifications@github.com>
发送时间: Friday, September 6, 2019 2:23:00 AM
收件人: philipperemy/keract <keract@noreply.github.com>
抄送: zkx741481546 <zkx741481546@live.com>; Author <author@noreply.github.com>
主题: Re: [philipperemy/keract] Add function: (#57)


LGTM but updating the docstrings to reflect the new parameters would be a good idea

―

**Comment by zkx741481546** [comment]
I had tried Plotly, but it's not faster than matlibplot, I am hoping for a very low performance loss that I can draw during training

**Comment by zkx741481546** [comment]
If user are not using keras, Keras.Backend wont work. But it seems a good choice for me

**Comment by qwertpi** [comment]
> If user are not using keras, Keras.Backend wont work. But it seems a good choice to me

If the user isn't using keras then the **Kera**s **Act**ivations + Gradients library won't work

**Comment by philipperemy** [comment]
How ready is this PR?

**Comment by philipperemy** [comment]
Updated here https://github.com/philipperemy/keract/pull/62 Thanks for contributing :)
